client/js/views/Home.vue

```diff
@@ -1,14 +1,20 @@
 <template>
-  <div class="home">
-    <div class="container">
+  <div class="home pa-4" :class="{
+    'home--large': !$vuetify.breakpoint.mdAndDown,
+  }">
+    <v-container>
+      <v-row v-if="$vuetify.breakpoint.mdAndDown">
+        <img id="logo" alt="Cuttle logo" src="../img/logo.png" height="20vh" class="mb-4" />
+      </v-row>
       <div id="game-list-card">
         <v-row>
-          <v-col cols="9">
-            <v-row id="card-content-header" class="mb-4">
-              <v-col cols="4">
-                <h1 id="home-card-title">Games</h1>
-              </v-col>
-            </v-row>
+          <v-col :cols="$vuetify.breakpoint.mdAndDown ? 12 : 9">
+            <div id="card-content-header" class="mb-4">
+              <h1 id="home-card-title">Games</h1>
+              <v-row  class="create-game--btn"> 
```

Comment:
Nitpicky, but it looks like we have an extra space here between `v-row` and `class`